### PR TITLE
complete tests for 09-get-api-users

### PR DESCRIPTION
**Pull request by alxjnr**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -216,4 +216,26 @@ describe("API Testing", () => {
...
+    describe("GET reqeusts", () => {
+      test("GET /api/users should respond with a status code of 200", () => {
+        return request(app).get("/api/users").expect(200);
+      });
+      test("GET /api/users should return an array of user objects", () => {
+        return request(app)
+          .get("/api/users")
+          .expect(200)
+          .then((res) => {
+            expect(typeof res.body.rows).toBe("object");
```
Comment (corkesw) [inline __tests__/app.test.js]: I would expect the data to come back on a key of users rather than rows

File: models/models.users.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const db = require("../db/connection");
+
+exports.fetchUsers = (s) => {
+  return db.query(`SELECT * FROM users`).then((res) => {
+    return res;
```
Comment (corkesw) [inline models/models.users.js]: We are only interested in the rows here, not the whole res object

File: controllers/controllers.users.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const { fetchUsers } = require("../models/models.users.js");
+
+exports.getUsers = (req, res, next) => {
+  fetchUsers().then((users) => {
+    res.status(200).send(users);
```
Comment (corkesw) [inline controllers/controllers.users.js]: We should be sending this back in an object on a key of users

**Review by corkesw — COMMENTED** [review]
Not quite right here, you should change the test assertion to expect your data to be on the correct key. You'll need to adjust your logic in the controller and model to pass this test.

File: controllers/controllers.users.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+const { fetchUsers } = require("../models/models.users.js");
+
+exports.getUsers = (req, res, next) => {
+  fetchUsers().then((users) => {
+    // console.log(users.rows);
```
Comment (lewispricey) [inline controllers/controllers.users.js]: c.log

File: __tests__/app.test.js [hunk 5]
```diff
@@ -216,4 +216,27 @@ describe("API Testing", () => {
...
+    describe("GET reqeusts", () => {
+      test("GET /api/users should respond with a status code of 200", () => {
+        return request(app).get("/api/users").expect(200);
+      });
+      test.only("GET /api/users should return an array of user objects", () => {
+        return request(app)
+          .get("/api/users")
+          .expect(200)
+          .then((res) => {
+            // console.log(res.body.users);
```
Comment (lewispricey) [inline __tests__/app.test.js]: c.log

**Review by lewispricey — APPROVED** [review]
Great work on this Alex, thanks for implementing Stephen's feedback the changes look good!

A few console logs to remove before merging and I'd recommend adding in an extra assertion in your test to check the users length isn't zero as that could cause false positives in the test but once done please feel free to merge 🚀
